src/mnode/mnode-messageproc.h

```diff
@@ -38,6 +47,7 @@ class CMasternodeMessage
     {
         READWRITE(vinMasternodeFrom);
         READWRITE(vinMasternodeTo);
+        READWRITE(messageType);
```

Comment:
probably need to move this to the end?
will be supported correct by old clients?